apps/discordbot/bot.py

```diff
@@ -22,6 +22,11 @@
 HOSER_ROLE_ID = config("TEST_ROLE", default=785418569412116513, cast=int)  # Verified
 DEBUG_CHANNEL_ID = config("DEBUG_CHANNEL", default=788989977794707456, cast=int)
 TIMEOUT_SECS = 10
+VALID_XKCD_COMMANDS = ["-random", "-r", "-help", "-h", "-issue", "-i"]
+XKCD_RANDOM_URL = "https://c.xkcd.com/random/comic/"
+XKCD_HELP_MSG = "```!xkcd help\n" + ("-" * 50) + "\n-help (-h)  /  Displays help for the '!xkcd' command.\n" \
+                "-random (-r)  /  Displays a random XKCD issue.\n-issue (-i) #  /  Displays a specific XKCD issue.```"
```

Comment:
Use a multi-line string here

```suggestion
XKCD_HELP_MSG = f"""```!xkcd help
{"-" * 50}
-help (-h)  /  Displays help for the '!xkcd' command.
-random (-r)  /  Displays a random XKCD issue.
-issue (-i) #  /  Displays a specific XKCD issue.```
```